docker-compose/.env

```
@@ -1,5 +1,77 @@
+SB_VERSION=4
+# Config directory mounted into the app. Copied from data.example/.
+SB_CONFIG_DIR=./data
+# Prefix for container and volume names. Change it to run two stacks side by side.
+# COMPOSE_PROJECT_NAME=suivi-bourse
+
+# ---------------------------------------------------------------------------
+# Ports
+# ---------------------------------------------------------------------------
+GRAFANA_PORT=3000
+INFLUXDB_PORT=8181
+GF_ADMIN_PASSWORD=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GF_ADMIN_PASSWORD key should go before the GRAFANA_PORT key

(UnorderedKey)